nulib/meadow
# Simplify MIME Type extraction

# Summary 
Simplify MIME Type extraction

# Specific Changes in this PR
- Remove XML parsing from MIME extractor (possibly to be addressed later by [repodev_planning_and_docs#2740](https://github.com/nulib/repodev_planning_and_docs/issues/2740)
- Trust file extensions
- Return `text/plain` for `.md5` and `.framemd5`
- Still fall back to `application/octet-stream` for S file types

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

Run various files through the ingest pipeline/MIME extraction lambda and make sure their MIME types are determined and recorded correctly.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
